### ISSUE TITLE
Error when attempting to map for a migration with CVSimp

### ISSUE THREAD

**Issue by comtineo01** [xtuple/xtuple#2128]
I'm an accountant, not an IT/Coder, so dumb it down for me please. IT guy insists it's not a problem with the database, but I can't find the issue to fix in the CVSimp tool for the life of me.

Getting this error when I try to map/import the customer or vendor api's.

ERROR Record 1: ERROR:  ReferenceError: XT is not defined
DETAIL:  uuid_generate_v4() LINE 7:   return XT.generateUUID();
QPSQL: Unable to create query

**Comment by andersdd** [comment]
This looks to be an issue with a mobilized database.

My suggestion is in your CSVImp tool open your Atlas Map (for example the Customer API map) and in the preSQL tab enter the following code:  select xt.js_init();

Save the map and the import should work.  You will need to do this for each map that has this problem.

**Comment by xtpclark** [comment]
For your pre-SQL, try adding 'SELECT xt.js_init();'
This function needs to be initialized before importing into particular tables.

- Perry Clark
xtuple

**Comment by xtpclark** [comment]
For your pre-SQL, try adding 'SELECT xt.js_init();'
This function needs to be initialized before importing into particular tables.

- Perry Clark
xtuple

**Comment by xtpclark** [comment]
For your pre-SQL, try adding 'SELECT xt.js_init();'
This function needs to be initialized before importing into particular tables.

- Perry Clark
xtuple

**Comment by xtpclark** [comment]
For your pre-SQL, try adding 'SELECT xt.js_init();'
This function needs to be initialized before importing into particular tables.

- Perry Clark
xtuple

**Comment by xtpclark** [comment]
For your pre-SQL, try adding 'SELECT xt.js_init();'
This function needs to be initialized before importing into particular tables.

- Perry Clark
xtuple

**Comment by xtpclark** [comment]
For your pre-SQL, try adding 'SELECT xt.js_init();'
This function needs to be initialized before importing into particular tables.

- Perry Clark
xtuple

**Comment by xtpclark** [comment]
For your pre-SQL, try adding 'SELECT xt.js_init();'
This function needs to be initialized before importing into particular tables.

- Perry Clark
xtuple

**Comment by xtpclark** [comment]
For your pre-SQL, try adding 'SELECT xt.js_init();'
This function needs to be initialized before importing into particular tables.

- Perry Clark
xtuple

**Comment by xtpclark** [comment]
For your pre-SQL, try adding 'SELECT xt.js_init();'
This function needs to be initialized before importing into particular tables.

- Perry Clark
xtuple

**Comment by xtpclark** [comment]
For your pre-SQL, try adding 'SELECT xt.js_init();'
This function needs to be initialized before importing into particular tables.

- Perry Clark
xtuple

**Comment by xtpclark** [comment]
For your pre-SQL, try adding 'SELECT xt.js_init();'
This function needs to be initialized before importing into particular tables.

- Perry Clark
xtuple

**Comment by xtpclark** [comment]
For your pre-SQL, try adding 'SELECT xt.js_init();'
This function needs to be initialized before importing into particular tables.

- Perry Clark
xtuple

**Comment by xtpclark** [comment]
For your pre-SQL, try adding 'SELECT xt.js_init();'
This function needs to be initialized before importing into particular tables.

- Perry Clark
xtuple

**Comment by xtpclark** [comment]
For your pre-SQL, try adding 'SELECT xt.js_init();'
This function needs to be initialized before importing into particular tables.

- Perry Clark
xtuple

**Comment by comtineo01** [comment]
Thank you VERY much!!
